### ISSUE TITLE
Bot did not work on TNF drop

### ISSUE THREAD

**Issue by something102347** [val92130/Supreme-Auto-Checkout#123]
I had the bot set up to and once it dropped the bot just kept refreshing and it didn't show any of the products just a white screen. I thought the site crashed but when i went onto a new tab it loaded fine. I tried the bot again and it just doesn't load the items and just gives a blank screen. Any idea why this happened?

**Comment by something102347** [comment]
If they changed the font on all products the autocop would not work. However if you try for yourself to use the bot it works for all items accept the TNF tee which is quite confusing.

**Comment by MuccaDiocan** [comment]
Maybe cause its OOS now so you can't really say it?

**Comment by something102347** [comment]
whats OOS?

**Comment by MuccaDiocan** [comment]
out of stock, if the item its out of stock it doesn't find it

**Comment by something102347** [comment]
Yeah i know that it buts thats not what i'm saying, even for the t-shirts, the page does not load. Look at the picture. (https://gyazo.com/446ed5bc54736497774d2d40e4f6950f) Its a blank screen which just refreshing. It only does this for the tee. For any other product even if out of stock, the page will still load the images and the items but there is some sort of problem with the tee.

**Comment by MuccaDiocan** [comment]
Oh yeah then I don't know I will quick try and let you know
